modules/persistent-storage-csi-migration-overview.adoc

```diff
@@ -6,30 +6,19 @@
 [id="persistent-storage-csi-migration-overview_{context}"]
 = Overview
 
-Volumes that are provisioned by using in-tree storage plugins, and that are supported by this feature, are migrated to their counterpart Container Storage Interface (CSI) drivers. This process does not perform any data migration; {product-title} only translates the persistent volume object in memory. As a result, the translated persistent volume object is not stored on disk, nor is its contents changed.
+This feature automatically migrates volumes that were provisioned using in-tree storage plugins to their counterpart Container Storage Interface (CSI) drivers. 
 
-The following in-tree to CSI drivers are supported:
+This process does not perform any data migration; {product-title} only translates the persistent volume object in memory. As a result, the translated persistent volume object is not stored on disk, nor is its contents changed. CSI automatic migration should be seamless. This feature does not change how you use all existing API objects: for example, `PersistentVolumes`, `PersistentVolumeClaims`, and `StorageClasses`.
 
-.CSI automatic migration feature supported in-tree/CSI drivers
-[cols=",^v,^v,^v width="100%",options="header"]
-|===
-|In-tree/CSI drivers |Support level |CSI auto migration enabled automatically?
+The following in-tree to CSI drivers are automatically migrated:
 
-a|
 * Azure Disk
 * OpenStack Cinder
 * Amazon Web Services (AWS) Elastic Block Storage (EBS)
 * Google Compute Engine Persistent Disk (GCP PD)
-|Generally available (GA)
-| Yes. For more information, see "_Automatic migration of in-tree volumes to CSI_".
-
-a|
 * Azure File
-* VMware vSphere
-|Technology Preview (TP)
-| No. To enable, see "_Manually enabling CSI automatic migration_".
-|===
+* VMware vSphere (see information below for specific migration behavior for vSphere)
```

Comment:
Sorry to add a comment here, @lpettyjo. As part of accessibility requirements, we should avoid the word "Below" in this way. Maybe add a hardcode reference to the next module?

> See "Sphere automatic migration" section.

IBM SG:

![Screenshot from 2023-05-09 11-34-43](https://github.com/openshift/openshift-docs/assets/57954076/5912583f-2c9e-4040-92f2-906049eb1db2)

Reply:
This looks like you should use the term "below" as it says "Use only...or Below.." Also, I cannot hardcode a reference (I assume that you mean xref) as we are not allowed to have xrefs in modules.